labcore/analysis/hvapps.py

```diff
@@ -32,9 +32,12 @@ def __init__(self, update_callback):
         self.update_callback = update_callback
 
     def on_created(self, event):
-        if event.is_directory:
-            self.update_callback(event)
-
+        if not event.is_directory:
+            # Get file extension and compare to data file type, ddh5
+            file_type = os.path.splitext(event.src_path)[1].lower()
+            #TODO: Generalize to other data file types
```

Comment:
use `pathlib` for this, not `os`: https://docs.python.org/3/library/pathlib.html